src/main/scala/com/sclasen/akka/kafka/AkkaHighLevelConsumer.scala

```diff
@@ -0,0 +1,177 @@
+                           receiver: ActorRef,
+                           msgHandler: (MessageAndMetadata[Key,Msg]) => Any = defaultHandler[Key, Msg],
+                           connectorActorName:Option[String] = None,
+                           maxInFlightPerStream: Int = 64,
+                           startTimeout: Timeout = Timeout(5 seconds),
+                           commitConfig: CommitConfig): AkkaConsumerProps[Key, Msg] =
+    AkkaConsumerProps(context.system, context, zkConnect, Right(topic), group, streams, keyDecoder, msgDecoder, msgHandler, receiver,connectorActorName, maxInFlightPerStream, startTimeout, commitConfig)
+
+  def forContextWithFilter[Key, Msg](context: ActorContext,
+                                     zkConnect: String,
+                                     topicFilter: TopicFilter,
+                                     group: String,
+                                     streams: Int,
+                                     keyDecoder: Decoder[Key],
+                                     msgDecoder: Decoder[Msg],
+                                     receiver: ActorRef,
+                                     msgHandler: (MessageAndMetadata[Key,Msg]) => Any = defaultHandler[Key, Msg],
+                                     connectorActorName:Option[String] = None,
+                                     maxInFlightPerStream: Int = 64,
+                                     startTimeout: Timeout = Timeout(5 seconds),
+                                     commitConfig: CommitConfig): AkkaConsumerProps[Key, Msg] =
+    AkkaConsumerProps(context.system, context, zkConnect, Left(topicFilter), group, streams, keyDecoder, msgDecoder, msgHandler, receiver,connectorActorName, maxInFlightPerStream, startTimeout, commitConfig)
+
+  def defaultHandler[Key,Msg]: (MessageAndMetadata[Key,Msg]) => Any = msg => msg.message()
+}
+
+case class AkkaHighLevelConsumerProps[Key,Msg](system:ActorSystem,
+                                               actorRefFactory:ActorRefFactory,
+                                               zkConnect:String,
+                                               topicFilterOrTopic:Either[TopicFilter,String],
+                                               group:String,
+                                               streams:Int,
+                                               keyDecoder:Decoder[Key],
+                                               msgDecoder:Decoder[Msg],
+                                               msgHandler: (MessageAndMetadata[Key,Msg]) => Any,
+                                               receiver: ActorRef,
+                                               connectorActorName:Option[String],
+                                               maxInFlightPerStream:Int = 64,
+                                               startTimeout:Timeout = Timeout(5 seconds),
+                                               commitConfig:CommitConfig = CommitConfig())
+
```

Comment:
maxinFlightPerStream and commitConfig should be removed, as they dont do anything in this connector.